src/main/java/hudson/plugins/setenv/SetEnvBuildWrapper.java

```diff
@@ -49,7 +49,7 @@ public Environment setUp(AbstractBuild build, Launcher launcher, BuildListener l
     public static final class DescriptorImpl extends BuildWrapperDescriptor {
         @Override
         public String getDisplayName() {
-            return null;
+            return "SetEnvBuildWrapper";
```

Comment:
If you put a name, these wrappers will start appearing in the lists IIRC

Reply:
Hmm i guess i could set it to a blank name so spotbugs wont complain, would that be ok?